docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: "3.8"
+
+services:
+  db:
+    image: 'postgres:13.1-alpine'
+    container_name: pg_db
+    env_file: .env
+    environment:
+      - POSTGRES_USER=${DOCKER_DB_USER}
+      - POSTGRES_PASSWORD=${DOCKER_DB_PASSWORD}
+      - POSTGRES_DB=${DOCKER_DB_NAME}
+    ports:
+      - "5432:5432"
+    volumes:
+      - db:/var/lib/postgresql/data
+    networks:
+      - network
+
+
+  backend:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: gpu-configurator-backend-dec-java
+    container_name: gpu-configurator-backend-dec-backend
+    restart: no
```

Comment:
I believe you could consider other restart policies other than "no" (you could use "on-failure") since developers can sometimes push code that may crash when you hit specific endpoints, and hence, the application should be able to restart on its own without manual intervention.

Reply:
alright thanks